### PR TITLE
Added documentation on Message class

### PR DESCRIPTION
**Pull request by FloweyTheFlower420**
Added documentation for public member variables in the Message class, so people don't have to scroll through to find the author of a message. There may be some issues with indention, but ill fix it later

### CODE REVIEW

File: source/documentation.html.md [hunk 1]
```diff
@@ -903,6 +903,131 @@ Message(BaseDiscordClient* client, std::string channelID, std::string message, b
...
+#### timestamp
+```cpp
+std::string timestamp;
+```
+
+When the message was sent, as a string.
+
+#### editedTimestamp
+```cpp
+std::sstring editedTimestamp;
```
Comment (yourWaifu) [inline source/documentation.html.md:945]: extra s

File: source/documentation.html.md [hunk 2]
```diff
@@ -903,6 +903,131 @@ Message(BaseDiscordClient* client, std::string channelID, std::string message, b
...
+```
+
+Whether or not the message mentions `@everyone`. The default value is `false`.
+
+#### mentions
+```cpp
+std::vector<User> mentions;
+```
+
+List of everyone mention in this message.
```
Comment (yourWaifu) [inline source/documentation.html.md:969]: remove everyone

File: source/documentation.html.md [hunk 3]
```diff
@@ -903,6 +903,131 @@ Message(BaseDiscordClient* client, std::string channelID, std::string message, b
...
+      RECIPIENT_REMOVE       = 2,
+      CALL                   = 3,
+      CHANNEL_NAME_CHANGE    = 4,
+      CHANNEL_ICON_CHANGE    = 5,
+      CHANNEL_PINNED_MESSAGE = 6,
+      GUILD_MEMBER_JOIN      = 7
+    } type = DEFAULT;
+```
+
+The origin of the message.
```
Comment (yourWaifu) [inline source/documentation.html.md:1027]: not sure why this is here.

**Review by yourWaifu — CHANGES_REQUESTED** [review]

**Comment by FloweyTheFlower420** [comment]
Hey sorry, im kinda a idiot, and i will be creating another pull request.

**Comment by FloweyTheFlower420** [comment]
wait i think i figured it out lol

**Review by yourWaifu — APPROVED** [review]

**Comment by yourWaifu** [comment]
Don't worry if changes don't show up yet, I still have to deploy it.

**Pull request merged**